examples/lifecycle/client.spec.ts

```diff
@@ -35,8 +36,8 @@ describe('lifecycle typed client', () => {
   })
 
   test('create_bare_optin', async () => {
-    const createResult = await client.create({ updatable: true, onCompleteAction: 'opt_in' })
-    expectType<undefined>(createResult.return)
+    const createResult = await client.create.bare({ updatable: true, onCompleteAction: 'opt_in' })
+    expectType<undefined | ABIReturn>(createResult.return)
```

Comment:
Because we don't map bare call responses, this is the actual type returned by the algokit-util-ts client

Reply:
We know it will be undefined, can we override for bare calls?

Reply:
we can add a type assertion for `AppCallTransactionResultOfType<undefined>` if you think that makes sense